vmm/src/vm.rs

```diff
@@ -1493,6 +1493,7 @@ pub fn test_vm() {
             VcpuExit::Debug => {}
             VcpuExit::Hlt => {
                 println!("HLT");
+                break;
```

Comment:
Looks good!
But if we really want this to be part of the unit testing, we should replace `#[allow(unused)]` with `#[test]` on line 1422.

Reply:
My local copy has `#[test]`. I didn't do it because I wasn't sure the CI can handle this test case. I can submit a PR for that if you want.

Reply:
The CI should always have VT-x enabled, so I think it's okay to enable this test.